src/components/forms/formr-part-a/View.tsx

```diff
@@ -31,6 +31,10 @@ const View = ({ canEdit, history }: IView) => {
             </p>
           </WarningCallout>
         )}
+        <h3 data-cy="dateSubmitted">
```

Comment:
If you write a CT targeting this h3 you will want to give it a unique name from the h3 on line 212.